routes/api.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
 $this->domain('api.' . ltrim(config('session.domain'), '.'))->group(function () {
-    $this->get('token', ['uses' => 'User\SessionTokenController@index'])->name('session');
+    $this->get('token', ['uses' => 'User\SessionTokenController@generate_token']);
+    $this->post('token/verify', ['uses' => 'User\SessionTokenController@verify_token']);
+
+    $this->get('user/{user}', ['uses' => 'User\UserApiController@get']);
```

Comment:
Generalnie tak, jasne. Niedługo (w weekend w samolocie?) usiąde do API może, bo to tutaj to tak na kolanie zrobiłem :P. Było dużo pomysłów w wątku, przeczytam/połączę je i napiszę jakiś post z "planem rozwoju".